frontend/controller/router.js

```diff
@@ -91,12 +90,6 @@ const router: any = new Router({
       meta: { title: L('Design System') }
       // beforeEnter: createEnterGuards(designGuard)
     },
-    {
-      path: '/bypass-ui',
-      component: BypassUI,
-      name: BypassUI.name,
-      meta: { title: L('Cypress - BypassUI') }
-    },
```

Comment:
You're right we don't need that file anymore, but I've kept it in the project for now in case we decide we need to restore it for whatever reason. Currently it's not being imported, so it doesn't appear in the generated bundle.

Reply:
Why don't we save this in `historical` folder instead of the current location?

Reply:
@Silver-IT 👍 Yes, good idea, feel free to move it in your PR after merging `master` 😄

Reply:
Got it. I will do.